### DISCUSSION TITLE
Disable smooth scrolling?

### DISCUSSION THREAD

**Discussion by luajaz** [TriliumNext/Trilium#2757] [Q&A]
Is there any way to disable smooth scrolling or increase it's velocity? There is a branch made to remove it but it's 3 years old.

The app is awesome, but I really dislike the smooth scrolling.

**Comment by dousha** [comment]
Hi. You can pass `--disable-smooth-scrolling` argument to disable smooth scrolling.

You can create `trilium-no-smooth-scroll.bat` alongside `trilium.exe` with the following content:

```bat
@echo off
start trilium.exe --disable-smooth-scrolling
```
